static/app/components/replays/replayContext.tsx

```diff
@@ -98,9 +98,9 @@ const ReplayPlayerContext = React.createContext<ReplayPlayerContextProps>({
   fastForwardSpeed: 0,
   initRoot: () => {},
   isPlaying: false,
+  isSkippingInactive: false,
   setCurrentTime: () => {},
   setSpeed: () => {},
-  skipInactive: false,
```

Comment:
Before we were renaming the context value when setting it as a prop: `isSkippingInactive={skipInactive}`... better to use the same name throughout